fares-data-build-tool/fdbt-netex-output
# test: consolidated s3.ts files

# Description

-   This PR consolidates the two s3.ts files in the fdbt-netex repo. These files were discovered while reviewing test coverage across the repo and contain the same functionality.

# Testing instructions

-   Pull down this branch, run tests and run the make generate-validate-all command to check that the code still behaves as it should.

# Type of change


# Checklist:
